router/src/health.rs

```diff
@@ -66,7 +66,7 @@ impl Health {
             adapter_index: 0,
             // Block 0 is reserved for health checks
             blocks: vec![0],
-            slots: (0..16).collect(),
+            slots: (0..self.shard_info().block_size).collect(),
```

Comment:
Nit, but can we use the property directly instead of going through a function call that adds extra indirection of function call?

Reply:
sure